### PR TITLE
Update some texts in the modal element

### PR DESCRIPTION
**Pull request by amieiro**
#### Problem

Update some texts in the modal element, so we get the same text we have in the single action. See https://github.com/GlotPress/gp-translation-helpers/issues/95 and https://github.com/GlotPress/gp-translation-helpers/pull/90.

### Old texts

image

### New texts

image

### Single action texts

image

#### Solution

Update some texts.

#### Testing instructions

Check the new texts in the modal element are the same we have in the single action.

### CODE REVIEW

**Comment by akirk** [comment]
Could you also add a button that closes the popover if you don't want to add a message (the X already does it but it's not obvious)?

**Comment by amieiro** [comment]
@akirk I have updated the text in the "Reject" button with "Reject with no Feedback", so when the user hasn't added any feedback, this is the button she will see.

image

When the user clicks on a checkbox and/or adds some feedback in the textarea, she will see the "Request changes" button.

image

I think this is the best approach, to show only one button with the action the user can execute. If we add 2 buttons with a checkbox, people are going to have problems thinking about the action each button is going to do. What do you think?

**Review by akirk — APPROVED** [review]
I like it!

File: js/reject-feedback.js [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
+			'<h3>Type (Optional)</h3>' +
 			getReasonList() +
 			'<div class="modal-comment">' +
-					'<label>Comment </label>' +
-					'<textarea name="modal_feedback_comment"></textarea>' +
+					'<h3><label for="modal_feedback_comment">Comment (Optional)</label></h3>' +
+					'<textarea id="modal_feedback_comment" name="modal_feedback_comment"></textarea>' +
 			'</div>' +
-			'<button id="modal-reject-btn" class="modal-btn gp-btn-style">Reject</button>' +
+			'<button id="modal-reject-btn" class="modal-btn gp-btn-style">Reject with no Feedback</button>' +
```
Comment (akirk) [inline js/reject-feedback.js:17]: Maybe rather?
```suggestion
			'<button id="modal-reject-btn" class="modal-btn gp-btn-style">Reject without Feedback</button>' +
```

**Pull request merged**